cli/slashingExport.go

```diff
@@ -36,13 +36,35 @@ func SlashingExportCmd(sedgeActions actions.SedgeActions) *cobra.Command {
 	)
 
 	cmd := &cobra.Command{
-		Use:   "slashing-export [flags]",
+		Use:   "slashing-export [validator]",
 		Short: "Export slashing protection data",
-		Long:  "Export slashing protection interchange data (EIP-3076). Validator is stopped if is currently running",
+		Long: `
+Export Slashing Protection Interchange Format (EIP-3076) data. This command assumes
+that the validator client container exists, stopped or not and that its database
+is already initialized. Take in mind that the validator client generates slashing
+protection data after some time running, so for some clients export slashing protection
+data just after start the client could produce some errors.
+
+This command stops the validator client during the exporting process due to the
+validator database being locked while it's running but leaves the validator client
+in the same state in which it was found. That means if the validator is running/stopped
+before the export, then the validator will be running/stopped after the command
+is executed, regardless of whether the export fails or not. To force a different
+behavior use --start-validator and --stop-validator flags.`,
+		Args: func(cmd *cobra.Command, args []string) error {
+			if len(args) != 1 {
+				return ErrInvalidNumberOfArguments
+			}
+			return nil
+		},
```

Comment:
Use Cobra's `MinimumNArgs(1)` instead of that function

Reply:
I used Cobra's `MinimunArgs(1)` at the start, but with the tests was impossible to test that case because I need an error to test with the `assert.ErrorIs(...)` function and `cobra` packages don't have a public error that I can use. Please see the `TestSlahingExport_ValidatorIsRequired` test case.